events-api/src/modules/tickets/tickets.module.ts

```diff
@@ -0,0 +1,14 @@
+import { HttpModule } from '@nestjs/axios';
+import { TicketsController } from './tickets.controller';
+import { TicketsService } from './tickets.service';
+import { DatabaseModule } from './../database/database.module';
+import { Module } from '@nestjs/common';
+import { PaymentService } from '../payment/payment.service';
+
+@Module({
+  imports: [DatabaseModule, HttpModule],
```

Comment:
You can configure the HttpModule's axios here (base url, auth token header, etc)